IronFoundry.Warden.Service/install.ps1

```diff
@@ -40,6 +146,22 @@ catch {
         }
     }
 }
+
+# Set restrictions on what the WardenUsers can access.
+$AclMap = [ordered] @{ 
```

Comment:
We should investigate locking down the entire system by default.  Since there are many directories on a standard Windows install that don't inherit permissions from their parent, we can write the script to enumerate any directories that aren't locked down and apply specific permissions to that directory.